src/server/handlers.ts

```diff
@@ -1171,9 +1171,7 @@ export const addHandlers = (connection: Connection, redirectConsole = true) => {
                                     label: name,
                                     detail: path,
                                     insertText: name,
-                                    kind: path.startsWith('mo:')
-                                        ? CompletionItemKind.Module
-                                        : CompletionItemKind.Class, // TODO: resolve actors, classes, etc.
+                                    kind: CompletionItemKind.Module,
```

Comment:
Is it still the case if we have the `path` with an `actor` inside? Not sure why it considered `Class` before

Reply:
Good catch, I think it was `Class` because there is no `actor` completion kind. But I still think `path.startsWith('mo:')` is incorrect way to decide kind, because path could be a library module, not necessarily `actor`.